edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/agentStart.sh

```diff
@@ -65,6 +52,9 @@ then
     chown -fR "${TARGET_UID}" "${backuppath}"
   fi
 
+  # add user to iotedge group
+  usermod -aG iotedge "${TARGET_UID}" >/dev/null
```

Comment:
This is running in the context of the container - do we have an iotedge group there?

Reply:
Yes, this should only be done in package preinst, not here.

Reply:
Fixed